src/main/java/seedu/duke/operationlist/FlightList.java

```diff
@@ -54,19 +53,30 @@ public void addOperation(String detail) {
         }
     }
 
-    //not done
     @Override
     public void deleteOperation(String detail) {
         boolean isFlightFound;
         try {
             checkCommandLength(detail.substring("flight delete".length()));
```

Comment:
flight delete is repeated a few times, can consider putting it as a constant variable